### DISCUSSION TITLE
-Weight Extrabold - style Italic has no effect

### DISCUSSION THREAD

**Discussion by a123likeyou** [ImageMagick/ImageMagick#2894] [Help]
What should I do to make the font bold, italics, underline, delete line，I saw the bold and italicized commands on the official website,
 Magick  convert   -size 500x500 xc:red  -font f -undercolor red  -pointsize 50 -weight ExtraBold -style Italic  -draw "fill blue text 40,100 'Copyright'"  d:\overlap_canvas.png
However, there is no effect after use, I don't know why, and have you found the use of underline and strikeoutThank you very much for your help

**Comment by urban-warrior** [comment]
See https://legacy.imagemagick.org/Usage/text/#pango_markup.
